### PR TITLE
Fix "Server: Unkown Command" issue

### PR DESCRIPTION
**Pull request by ghost**
Fix "Server: Unkown Command" issue in /customtax command with some Edit :DD

### CODE REVIEW

File: pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn [hunk 1]
```diff
@@ -1016,7 +1016,11 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
 
         // Does the player own the airport?
         if (endid != playerid)
-            return 0;
+        {
```
Comment (RussellLVP) [inline pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn]: The brackets are not necessary. (Neither here nor on line 1022.)

File: pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn [hunk 2]
```diff
@@ -1016,7 +1016,11 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
...
+            SendClientMessage(playerid, COLOR_ORANGE, "Error: Sorry, You Don't Own The LV Airport"); //Fix "Server: Unkown Command" issue
```
Comment (RussellLVP) [inline pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn]: Please Don't Begin Each Word With A Capital.

That's not how the English language works. We require proper English grammer in the messages shown on Las Venturas Playground. What about:

> Error: Sorry, you can only use this command when you own the LV Airport!

File: pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn [hunk 3]
```diff
@@ -1016,7 +1016,11 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
...
+            SendClientMessage(playerid, Color::Green, "use /taxi 2 to be teleported to LV Airport");
```
Comment (RussellLVP) [inline pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn]: Suggestion:

> You can use "/taxi 2" to immediately teleport to the airport.

File: pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn [hunk 4]
```diff
@@ -1016,7 +1016,11 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
...
+        }
+        return 0;
```
Comment (RussellLVP) [inline pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn]: So, this is the bit that actually causes the "Server: Unknown command" message— returning `1` means that the command could be handled, returning `0` means that it couldn't.

In this case, the player will receive a message about not owning the airport, so the command has been handled. This means that you should return `1`.

File: pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn [hunk 5]
```diff
@@ -1028,7 +1032,7 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
 
         if(!tmp[0])
         {
-            SendClientMessage(playerid, Color::White, "Use: /customtax [amount]");
+            SendClientMessage(playerid, Color::Green, "Use: /customtax [amount]");
```
Comment (RussellLVP) [inline pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn:1034]: Most of our _usage_ messages are white. Why did you change the colour?

Comment (ghost) [inline pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn:1034]: Green Better ( White some players don't see it " Bad Graphics " )

Errors : Segmentation fault (core dumped)

I Hate That Errors ( I Don't know how to fix it )

**Comment by ghost** [comment]
close this is nub
